libs/platform/src/index.ts

```diff
@@ -11,3 +11,5 @@ export * from './lib/components/select/public_api';
 export * from './lib/components/split-menu-button/public_api';
 export * from './lib/domain/public_api';
 export * from './lib/components/info-label/public_api';
+export * from './lib/components/split-menu-button/public_api';
```

Comment:
remove split-menu-button, it is already there at line 11

Reply:
done